cpp/src/arrow/engine/substrait/serde.h

```diff
@@ -36,193 +36,193 @@ namespace arrow {
 namespace engine {
 
 /// Factory function type for generating the node that consumes the batches produced by
-/// each toplevel Substrait relation when deserializing a Substrait Plan.
+/// each top-level Substrait relation when deserializing a Substrait Plan.
 using ConsumerFactory = std::function<std::shared_ptr<compute::SinkNodeConsumer>()>;
 
-/// \brief Deserializes a Substrait Plan message to a list of ExecNode declarations
+/// \brief Deserializes a Substrait Plan message to a list of ExecNode declarations.
```

Comment:
While we are at it, can we follow the convention of using the infinitive in function summaries.
```suggestion
/// \brief Deserialize a Substrait Plan message to a list of ExecNode declarations.
```

Reply:
6d61c85